### PR TITLE
Update deinterlacing methods to current best practices

### PR DESCRIPTION
**Pull request by MobyGamer**
For many years, hardware devices and online streaming platforms have been capable of 50p or 60p output, so the older advice of deinterlacing 25i/30i to 25p/30p is no longer necessary.  For matching the fidelity of interlaced sources as they were originally viewed on period hardware, each field should be output to its own frame, producing 50p content from 25i sources, and 60p content from 30i sources.  These updates to ffmprovisr reflect that.

Additionally, the deinterlacing filter advice of yadif or w3fdif has been changed to bwdif, which combines the best attributes of both older filters.

## Checklist

* [X] I've referred to the [Guidelines for contributing](https://github.com/amiaopensource/ffmprovisr/blob/gh-pages/readme.md#guidelines-for-contributing)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1354,15 +1354,17 @@ <h2 id="interlacing">Work with interlaced video</h2>
...
         <dt>ffmpeg</dt><dd>starts the command</dd>
         <dt>-i <em>input_file</em></dt><dd>path, name and extension of the input file</dd>
         <dt>-c:v libx264</dt><dd>encodes video stream with libx264 (h264)</dd>
         <dt>-filter:v</dt><dd>a video filter will be used</dd>
         <dt>"</dt><dd>quotation mark to start filtergraph</dd>
-        <dt>yadif</dt><dd>deinterlacing filter (‘yet another deinterlacing filter’)<br>
-         By default, <a href="https://ffmpeg.org/ffmpeg-filters.html#yadif-1" target="_blank">yadif</a> will output one frame for each frame. Outputting one frame for each <em>field</em> (thereby doubling the frame rate) with <code>yadif=1</code> may produce visually better results.</dd>
+        <dt>idet</dt><dd>detect interlaced video field order<br>
+         <a href="https://ffmpeg.org/ffmpeg-filters.html#idet" target="_blank">idet</a> will try to detect if the video is interlaced, and if so, what the order of the fields are (top-field-first, or bottom-field-first).  This is done to ensure the output of the deinterlacing filter is correct.</dd>
+        <dt>bwdif</dt><dd>deinterlacing filter (‘Bob Weaver Deinterlacing Filter’)<br>
```
Comment (ablwr) [inline index.html]: Haha, I didn't know this was named after a dude

**Review by ablwr — APPROVED** [review]
Lovely, thorough, insightful, and very welcome. Thank you! I'll approve this and allow some time for fellow archivists and maintainers to weigh in.

**Comment by kieranjol** [comment]
I echo Ashley, this looks really great!
I’d like to take a little more time later to review this thread https://lsv.uky.edu/scripts/wa.exe?A2=2304&L=AMIA-L&D=0&P=51537370 with regards to 50/60fps as best practice. Off the top of my head, still giving people the knowledge and option of both methods ( doubling vs not doubling) may be a way forward, and I’d love to hear from others!

**Comment by privatezero** [comment]
I wouldn't say that AMIA thread noted by @kieranjol resolved in any consensus around best practices. I think this is a great command, but my preference would be to have as a separate recipe rather than as an update of an existing one as that way ffmprovisr gains explanations of more filters/options and the ultimate choice is up to the user.

**Comment by kieranjol** [comment]
I agree Andrew!

**Comment by retokromer** [comment]
I second @privatezero’s position: another command would be better than simply eliminating `yadif`.

**Comment by MobyGamer** [comment]
> I think this is a great command, but my preference would be to have as a separate recipe rather than as an update of an existing one as that way ffmprovisr gains explanations of more filters/options and the ultimate choice is up to the user.

I can make those changes and resubmit, although I'm not sure what the correct process is.  Is it to reject/close this pull request, then submit a future one with the suggested changes?

**Comment by ablwr** [comment]
Honestly do whatever is easiest for you :)

**Comment by ablwr** [comment]
to extend that sentiment just a bit: I don't know how you're engaging with git/GitHub, but one method is that you could go into this branch you are working on, make changes and a new commit, and push those new commits to your branch and it will update here (you can also do a "force push" using -f to try to overwrite your existing work, if that's what you want to do) ~or~ you can close and re-open a new one, which is sometimes the easier thing to do, especially if you don't spend your time in git every day.

**Comment by MobyGamer** [comment]
I wasn't aware that pull requests could be automatically updated.  I'll attempt updating the branch, and will notify again if that appeared to be successful.

**Comment by ablwr** [comment]
Yep, they'll auto-update here!

**Comment by MobyGamer** [comment]
I reverted the previous sections and added a new section, "deinterlace fields to frames".  I also included an animated example showing the result of such deinterlacing (which, IMO, should be the default for all born-interlaced content).  If these changes and the new graphics are not visible, please let me know.

**Review by retokromer — APPROVED** [review]

**Review by ablwr — APPROVED** [review]
Love the additional pics too

**Comment by ablwr** [comment]
@privatezero and/or @kieranjol please review and feel free to merge!

**Review by kieranjol — APPROVED** [review]

**Comment by kieranjol** [comment]
It's perfect, thank you for taking on board the feedback! The gif is so helpful too.

**Review by privatezero — APPROVED** [review]
Guess that  leaves me to merge! Thanks for the contribution!

**Comment by kfrn** [comment]
Late to the party but thanks for this, @MobyGamer!

**Pull request merged**